recipes/trio-asyncio/LICENSE.MIT

```diff
@@ -0,0 +1,20 @@
+The MIT License (MIT)
```

Comment:
So this file can be dropped. `license_file` picks up the license from the source.

Reply:
The license isn't deployed with the pip install command therefore I initially dropped it but was then asked to include a copy.

Reply:
This is strange since the `license_file` does not appear to be in the tarball from pypi and I don't see any mention of it in the CI logs.

Reply:
Explained by new feature of conda-build https://github.com/conda-forge/staged-recipes/issues/5749